lib/lintMd.js

```diff
@@ -16,6 +16,12 @@ module.exports = function(from) {
             line_length: {
                 tables: false,
                 headings: false
+            },
+            MD007: {
+                indent: 4
+            },
+            MD013: {
+                headings: false
```

Comment:
Show me some examples where this is breaking.
@Kristiyan-Todorov was having an issue with the same, but I think he has found a better way.
Generally headers should not contain a lot of text.